scripts/update_nolints_CI.sh

```diff
@@ -32,9 +32,9 @@ gh_api() {
     "https://api.github.com/$url" "$@"
 }
 
-git push "${remote_name}" "HEAD:$branch_name"
+git push -v "${remote_name}" "HEAD:$branch_name"
```

Comment:
Is this change needed? It probably doesn't hurt, though.

Reply:
Yeah, I think you may have added it for debugging purposes at the beginning of this branch. I'll clean it up in another commit. (I want to make the `git fetch` step less noisy since it's currently spamming the log with a line for every open branch in this repo)